CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [CHANGE] Distributor: removed previously deprecated `extend_writes` (see #1856) YAML key and `-distributor.extend-writes` CLI flag from the distributor config. #2551
 * [CHANGE] Ingester: removed previously deprecated `active_series_custom_trackers` (see #1188) YAML key from the ingester config. #2552
 * [CHANGE] The tenant ID `__mimir_cluster` is reserved by Mimir and not allowed to store metrics. #2643
+* [CHANGE] Purger/Compactor: remove the purger component and move endpoints `/purger/delete_tenant` and `/purger/delete_tenant_status` to the compactor, both now `/compactor/delete_tenant...`. #2644
```

Comment:
Reviving a previous comment, since wasn't addressed.

```suggestion
* [CHANGE] Purger: removed the purger component and moved its API endpoints `/purger/delete_tenant` and `/purger/delete_tenant_status` to the compactor at `/compactor/delete_tenant` and `/compactor/delete_tenant_status`. #2644
```